backend/ohq/management/commands/membership_stat.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from ohq.models import Membership, Question
+from ohq.statistics import calculate_instructor_avg_time_stats, calculate_student_avg_time_stats
+
+
+class Command(BaseCommand):
+    def calculate_statistics(self, users):
+
+        # Iterate through all the users, building membership stats for each user
+        for user in users:
+            memberships = Membership.objects.filter(user=user, course__archived=False)
+            for membership in memberships:
+                questions_asked = Question.objects.filter(
+                    asked_by=user, status=Question.STATUS_ANSWERED, queue__course=membership.course
+                )
+                questions_answered = Question.objects.filter(
+                    responded_to_by=user,
+                    status=Question.STATUS_ANSWERED,
+                    queue__course=membership.course,
+                )
+                if questions_asked:
```

Comment:
Maybe do the conditional based on the kind of membership (student vs. course staff). Then, we can do the query on Questions based on the membership type instead of having an extra query that we don't ever use.

Reply:
Done